src/chkstat.h

```diff
@@ -88,6 +91,17 @@ class Chkstat {
      **/
     bool m_apply_changes = false;
 
+    /// Allow operation without mounted /proc file system.
+    /**
+     * In normal operation we fall back to a read only mode if /proc is not
+     * available because securely operating on arbitrary filesystems is not
+     * possible then. In some configurations /proc is not available (e.g.
+     * container setups) and the user choses to apply changes. The user
+     * needs to ensure that no unpriviled user can influence the operation
+     * in nefatious ways
```

Comment:
You probably mean `nefarious`.

I would change `and the user choses to apply changes` to `but changes should be applied anyway`.

Also `The caller needs to ensure` sounds better to me. 

Also you can declare this variable `const` which gives us the guarantee that the setting never accidentally changes after evaluating the environment variable during construction time.